scripts/build/publish_test.go

```diff
@@ -1,110 +0,0 @@
-package main
```

Comment:
Removing this file seems weird... Its something we want to keep.

Reply:
@bergquist well, `scripts/build/publish.go` was deleted [here](https://github.com/grafana/grafana/commit/55f2495afa043b67f9fc567d59c58eccc5d767c8). Why do we want to keep it?

Reply:
Oh sorry. didn't catch that.